ERCS/erc-7769.md

```diff
@@ -160,6 +159,22 @@ Response:
 }
 ```
 
+##### Support for [EIP-7702](./eip-7702.md) authorizations
+
+On networks with [EIP-7702](./eip-7702) activated, the `UserOperation` object may also contain an `eip7702Auth` tuple.
+When providing the `eip7702Auth` tuple, the fields `factory` and `factoryData` have a modified behaviour.
+
+The `factory` field must be set to exactly the `INITCODE_EIP7702_MARKER = 0x7702` flag.
+Passing his flag instructs the `EntryPoint` contract to verify the `sender` address contains a valid EIP-7702 authorization.
+
+The `factoryData` value is passed directly to the `sender` contract instead of the `factory` contract.
+This is done in a separate frame, meaning the `sender` contract will be called two times during validation.
```

Comment:
The term frame was never used in 4337, only 7560.
It is a call just before calling validateUserOp

Reply:
Rephrased this as well.